lib/HLSL/DxilContainerReflection.cpp

```diff
@@ -2043,9 +2047,7 @@ void DxilShaderReflection::CreateReflectionObjectsForSignature(
     D3D12_SIGNATURE_PARAMETER_DESC Desc;
     Desc.ComponentType = CompTypeToRegisterComponentType(SigElem->GetCompType());
     Desc.Mask = SigElem->GetColsAsMask();
-    // D3D11_43 does not have MinPrecison.
-    if (m_PublicAPI != PublicAPI::D3D11_43)
-      Desc.MinPrecision = CompTypeToMinPrecision(SigElem->GetCompType());
+    Desc.MinPrecision = CompTypeToMinPrecision(SigElem->GetCompType());
```

Comment:
The comment says D3D11_43 doesn't have min precision. So then, what will this be assigned to in the case that D3D11_43 is present? Or, will this code never be reached in that case?

Reply:
Our internal Desc is the full `D3D12_SIGNATURE_PARAMETER_DESC` that has min precision, so there's no harm in capturing it in the first place.  The code before was simply leaving that field uninitialized.  Combined with the interface bugs or the later bugs in `Get*ParameterDesc` functions, You could get uninitialized data from this stack copied out to your user struct in a later API call.

The code that copies the struct out to the user's struct is the code that needs to be careful about what it copies, hence the fixes in that code later.  Subtracting the `sizeof(D3D_MIN_PRECISION)` wasn't correct there because of the alignment of the struct size.